business-registry-dashboard/app/components/Form/AddBusiness/Base.vue

```diff
@@ -26,7 +26,7 @@ const props = defineProps<{
 }>()
 
 const emit = defineEmits<{
-  businessError: [{ error: FetchError, type: string }]
+  businessError: [{ error: FetchError, type: 'email' | 'delegation' | 'firm' | 'passcode' }]
```

Comment:
Added different error messages for firm or passcode type, previously showed the same for both, I overlooked this